src/ntid.ts

```diff
@@ -22,21 +22,21 @@ import { v4 as uuidv4 } from 'uuid';
 export type ntid = string;
 
 const UUID_LENGTH = 22;
-const UUID_ALPHABET =
-  'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_';
+const UUID_ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_';
```

Comment:
reading this over today I thought we should rename this since "UUID" refers to something different. Could just do "ID_BODY_LENGTH" and "ID_ALPHABET".

Reply:
Will update in the PR that codifies it as base64url (two up in the stack)